packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -224,9 +224,9 @@ contract GasPriceOracle is ISemver {
         uint256 operatorConstant = IL1Block(Predeploys.L1_BLOCK_ATTRIBUTES).operatorFeeConstant();
 
         if (isJovian) {
-            return Arithmetic.saturatingAdd(Arithmetic.saturatingMul(_gasUsed, operatorScalar) * 100, operatorConstant);
+            return _gasUsed * operatorScalar * 100 + operatorConstant;
```

Comment:
Could we / should we actually just restrict the data type for `_gasUsed` to a uint64?

Reply:
It will break the ABI so all downstream tooling will need to be updated, I don't think that is a good idea

Reply:
To clarify, changing the API like so would break the API:


```solidity
function getOperatorFee(uint256 _gasUsed) public;
function getOperatorFee(uint64 _gasUsed) public;
```

Doing something like this could work:

```
function getOperatorFee(uint256 _gasUsed) public {
  if (_gasUsed > type(uint64).max) revert OutOfBounds();
}
```

I am not super opinionated here, if we wanted to go with the explicit error message that would be nice but also its a very esoteric edge case and I think just letting the math revert is fine for keeping things more simple